rover-apps/common/src/CANModule.cpp

```diff
@@ -0,0 +1,21 @@
+#include "CANModule.h"
+
+#include "Module.h"
+#include "mbed.h"
+
+// Using initializer lists to construct CANInterface object will attach ISRs and other required tasks
+CANModule::CANModule(const Config &config) : interface(config) {
+    // Nothing should be required here
+}
+
+void CANModule::periodic_1ms(void) {
+
+    // These functions need to be called at 1 kHz, or every 1ms
+    interface.rxClient();
+}
+
+void CANModule::periodic_10ms(void) {
```

Comment:
was there a discussion on tx being 10ms?

Reply:
RX runs at 1kHz and TX runs at 100Hz

Reply:
i more meant like was there a reason those numbers were chosen? its fine if there wasn't- they seem reasonable anyways

Reply:
Hmm there's no concrete reason, other than we want RX to run faster than TX